app/javascript/components/ProgressPie.tsx

```diff
@@ -1,39 +1,45 @@
-import cx from "classnames";
 import * as React from "react";
 
-import { Progress } from "$app/components/Progress";
+import { classNames } from "$app/utils/classNames";
 
-export const ProgressPie = ({ progress, className }: { progress: number; className?: string }) => {
+import { Icon } from "$app/components/Icons";
+
+declare module "react" {
+  export interface CSSProperties {
+    "--progress"?: number | string;
+  }
+}
+
+export const ProgressPie = ({
+  progress,
+  className,
+  ...props
+}: { progress: number } & React.HTMLAttributes<HTMLDivElement>) => {
   const radius = 0.5;
-  const strokeWidth = 0.03;
```

Comment:
Kept the existing SVG logic from the old design here - was quite convenient! Just simplified it a little to remove the handling for strokes as we don't use one anymore.